app/saqib/api/todo/[id]/route.js

```diff
@@ -0,0 +1,22 @@
+// import { todos } from "../../../api/todo/route";
```

Comment:
instead of importing todos, you should try to do `var db = require('../database');` and then have a `database.js` file where you do all the updates